cmd/okctl/hooks/state.go

```diff
@@ -230,6 +230,29 @@ func PurgeRemoteState(o *okctl.Okctl) RunEer {
 	}
 }
 
+// VerifyClusterExistsInState ensures we have a cluster in the state before continuing
+// This could happen if maintenance mode was not run after upgrading
+// to 0.0.80 and we don't have a state.db in S3 - see #491
+func VerifyClusterExistsInState(o *okctl.Okctl) RunEer {
+	return func(_ *cobra.Command, _ []string) error {
+		handlers := o.StateHandlers(o.StateNodes())
+
+		_, err := handlers.Cluster.GetCluster(o.Declaration.Metadata.Name)
+		if err != nil {
+			log := logging.GetLogger(logComponent, "VerifyClusterExistsInState")
+			log.Debug("Could not find cluster in current state")
+
+			if errors.Is(err, storm.ErrNotFound) {
+				return fmt.Errorf("getting existing cluster %s: %w", o.Declaration.Metadata.Name, err)
```

Comment:
To be more specific, I would say `getting state for existing cluster %s: %w`, to emphasize that it's the state that's missing.